front/pages/w/[wId]/data-sources/vaults/[vaultId]/categories/[category]/index.tsx

```diff
@@ -57,6 +59,7 @@ export const getServerSideProps = withDefaultUserAuthRequirements<
       gaTrackingId: config.getGaTrackingId(),
       isAdmin,
       isBuilder,
+      canWrite,
```

Comment:
Outside of the server code let's be a bit more explicit
```suggestion
      canWriteInVault,
```

Reply:
Renamed everywhere!

Reply:
👌